src/khoj/processor/speech/text_to_speech.py

```diff
@@ -0,0 +1,51 @@
+import json  # Used for working with JSON data
+import os
+
+import requests  # Used for making HTTP requests
+from bs4 import BeautifulSoup
+from markdown_it import MarkdownIt
+
+# Define constants for the script
+CHUNK_SIZE = 1024  # Size of chunks to read/write at a time
+XI_API_KEY = os.getenv("ELEVEN_LABS_API_KEY", None)  # Your API key for authentication
+VOICE_ID = "29vD33N1CtxCmqQRPOHJ"  # ID of the voice model to use. MALE.
```

Comment:
We could expose a list of `VOICE_ID`s that the user can configure in their settings, similar to how we configure chat models (eventually).